portfolio/src/main/webapp/script.js

```diff
@@ -28,21 +28,35 @@ function addRandomFact() {
 }
 
 /**
- * Controls slideshow image display on the page
+ * Adds a new Comment Box to the page
+ */
+function newCommentBox(comment){
+  const commentBox = document.createElement("p");
+  commentBox.id = "comment"
```

Comment:
Do you need to give this element an explicit ID? This means every comment box will have the same ID, which is probably not what you want.